<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Program_Loop</class>
 <widget class="QWidget" name="Program_Loop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>26</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pgm Cycle</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QLabel" name="choose_program_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pgm Numb:</string>
    </property>
   </widget>
   <widget class="QLabel" name="all_loops_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cycle：</string>
    </property>
   </widget>
   <widget class="QLabel" name="program_name_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pgm Name：</string>
    </property>
   </widget>
   <widget class="QLabel" name="link_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Link：</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_program_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>270</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="all_loops_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>270</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="program_name_line">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>270</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="link_line">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>270</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="table_box" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>904</width>
      <height>284</height>
     </rect>
    </property>
    <widget class="QLabel" name="r1h0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>225</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Num.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="r2h0">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>72</y>
       <width>225</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Sgm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="r3h0">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>142</y>
       <width>225</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>End Sgm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="r4h0">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>212</y>
       <width>225</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r1h2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r2h2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r3h2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r4h2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r1h3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r2h3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>70</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r3h3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>140</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r4h3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>210</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r1h4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>0</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r2h4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>70</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r3h4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>140</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r4h4">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>210</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border_left">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>1</width>
       <height>283</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_right">
     <property name="geometry">
      <rect>
       <x>901</x>
       <y>2</y>
       <width>1</width>
       <height>283</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_top">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>902</width>
       <height>1</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_buttom">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>281</y>
       <width>902</width>
       <height>1</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="r1h1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r2h1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r3h1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="r4h1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>161</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>r1h1</zorder>
    <zorder>r1h0</zorder>
    <zorder>r2h0</zorder>
    <zorder>r3h0</zorder>
    <zorder>r4h0</zorder>
    <zorder>r1h2</zorder>
    <zorder>r2h2</zorder>
    <zorder>r3h2</zorder>
    <zorder>r4h2</zorder>
    <zorder>r1h3</zorder>
    <zorder>r2h3</zorder>
    <zorder>r3h3</zorder>
    <zorder>r4h3</zorder>
    <zorder>r1h4</zorder>
    <zorder>r2h4</zorder>
    <zorder>r3h4</zorder>
    <zorder>r4h4</zorder>
    <zorder>border_left</zorder>
    <zorder>border_right</zorder>
    <zorder>border_top</zorder>
    <zorder>border_buttom</zorder>
    <zorder>r2h1</zorder>
    <zorder>r3h1</zorder>
    <zorder>r4h1</zorder>
   </widget>
   <widget class="QLineEdit" name="choose_program_edit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="all_loops_edit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="program_name_edit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>110</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_edit">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>220</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* 正常状态 */
	border:2px solid rgb(74,122,60);
	background-color:rgb(173,199,160);
	border-radius:6px;
    color:rgb(74,122,60);
}

QPushButton:hover {
    /* 鼠标停留状态 */
    background-color: rgb(184, 192, 193);
}

QPushButton:pressed {
    /* 鼠标按下状态 */

    background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saving_pbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* 正常状态 */
	border:2px solid rgb(74,122,60);
	background-color:rgb(173,199,160);
	border-radius:6px;
    color:rgb(74,122,60);
}

QPushButton:hover {
    /* 鼠标停留状态 */
    background-color: rgb(184, 192, 193);
}

QPushButton:pressed {
    /* 鼠标按下状态 */

    background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_page_pbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* 正常状态 */
	border:2px solid rgb(74,122,60);
	background-color:rgb(173,199,160);
	border-radius:6px;
    color:rgb(74,122,60);
}

QPushButton:hover {
    /* 鼠标停留状态 */
    background-color: rgb(184, 192, 193);
}

QPushButton:pressed {
    /* 鼠标按下状态 */

    background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>LastPage</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="next_page_pbtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    /* 正常状态 */
	border:2px solid rgb(74,122,60);
	background-color:rgb(173,199,160);
	border-radius:6px;
    color:rgb(74,122,60);
}

QPushButton:hover {
    /* 鼠标停留状态 */
    background-color: rgb(184, 192, 193);
}

QPushButton:pressed {
    /* 鼠标按下状态 */

    background-color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>NextPage</string>
    </property>
   </widget>
   <widget class="QWidget" name="status_box" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="status_box_line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>1</y>
       <width>2</width>
       <height>59</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>15</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>One-Point</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="status_1">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>15</y>
       <width>25</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>15</y>
       <width>22</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>15</y>
       <width>15</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_4">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>18</y>
       <width>30</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>16</y>
       <width>27</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>15</y>
       <width>30</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
